### PR TITLE
main.cpp with file reading, 2 input test files

### PR DESCRIPTION
**Pull request by saif9817**
The first requested change to add file reading for the input, added an extra portion to default to a certain filename if no name is provided, have provided 2 test input files with a single change in the 2nd input, there are print statements commented out that were used for testing the functionality of the default and provided filename feature

### CODE REVIEW

File: top_model/top_model/main.cpp [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	//default filename if none is provided
+	if(argc<2){
+		inputReader.open("input_data/input.txt");
+	}else{
+		string temp = argv[1];
+		string filePath = "input_data/" + temp;
+		inputReader.open(filePath);
+	}
+	
+	if(inputReader.is_open()){
```
Comment (cruizm) [inline top_model/top_model/main.cpp:88]: Hello Saif,
This is good. Can you please modify it in such a way that in the name of the file we have the name of the input parameter, so everything is less error-prone?
Example:
mortality = 10.5
infectivity period = 15
...

**Review by cruizm — CHANGES_REQUESTED** [review]

**Comment by saif9817** [comment]
I have added the functionality to parse through tags, choosing to leave it in the exact order, I have added the assert to mortality only for testing, if you run the program with data.txt (which has an empty mortality field) it aborts the program but the message is not showing for me

**Comment by saif9817** [comment]
I have fixed the earlier issue, it is now providing specific errors

**Comment by saif9817** [comment]
Hi Cristina, please find the newly added comments for the code as well as a readme for the user input section and a template for users, please let me know if you need me to make any other changes

**Review by cruizm — APPROVED** [review]

**Pull request merged**